lib/app/services/api_network.dart

```diff
@@ -36,7 +30,7 @@ class ApiNetworkService {
     Map<String, String> parameters = const {},
   }) {
     return _network.get(
-      _url(path),
+      scWebUrl(path),
```

Comment:
`scWebUrl` is for the web-client, not for the API. Please also change the three occurrences below